src/dist/replication/meta_server/meta_bulk_load_service.cpp

```diff
@@ -1144,7 +1159,26 @@ void bulk_load_service::on_control_bulk_load(control_bulk_load_rpc rpc)
                                    false));
     } break;
     case bulk_load_control_type::BLC_RESTART:
-    // TODO(heyuchen): TBD
+        if (app_status != bulk_load_status::BLS_PAUSED) {
+            derror_f(
+                "restart bulk load for app({}) failed, can not restart bulk load with status({})",
+                app_name,
+                dsn::enum_to_string(app_status));
+            response.err = ERR_INVALID_STATE;
+            response.__set_hint_msg(
+                fmt::format("can not restart bulk load for app({}) with status({})",
+                            app_name,
+                            dsn::enum_to_string(app_status)));
```

Comment:
```suggestion
            auto hint_msg = fmt::format("can not restart bulk load for app({}) with status({})",
                            app_name,
                            dsn::enum_to_string(app_status)));
            derror(hint_msg.c_str());
            response.err = ERR_INVALID_STATE;
            response.__set_hint_msg(std::move(hint_msg);
```

Reply:
I think it is necessary for server to print error log of control command, I remain printing log here.